packages/@react-aria/slider/src/useSlider.ts

```diff
@@ -182,7 +184,8 @@ export function useSlider(
     }, moveProps),
     outputProps: {
       htmlFor: state.values.map((_, index) => getSliderThumbId(state, index)).join(' '),
-      'aria-labelledby': labelProps.id,
+      id: outputId,
+      'aria-labelledby': `${labelProps.id} ${outputId}`,
```

Comment:
Should there be logic here to handle if no label was passed in but an `aria-label` or `aria-labelledby` was provided in the props? I just noticed that the "Slider (hooks) -> with aria label stories" don't announce the label when moving the accessibility cursor to the output

would something like ``` 'aria-labelledby': `${labelProps.id || fieldProps.id} ${outputId}` ``` be appropriate?